Documentation/special-services/pod-mutator.md

```diff
@@ -0,0 +1,76 @@
+# PodMutator Service
+To provide pod mutation capabilities for finer-grained control over rkt containerization concepts and debugging needs, 
+this proposal introduces a new optional stage1 app/service with special semantics that is lightweight.
```

Comment:
I was confused by the term `app/service`, can we simply define the target annotation, i.e. `coreos.com/rkt/stage1/mutator` (or `coreos.com/rkt/stage1/manager` once we decide on the naming)